ros_installation.md

```diff
@@ -1,46 +1,97 @@
-# Install Gazebo and ROS
+# Installing Gazebo with ROS
 
-This document provides an overview about the options to use different versions
-of ROS in combination with different versions of Gazebo. It is recommended to read
+This document provides guidance on using different versions
+of ROS in combination with different versions of Gazebo. We recommended to read
 it before installing [ros_gz](https://github.com/gazebosim/ros_gz).
 
-# Important! Simple analysis for a quick and correct decision
+# Picking the "Correct" Versions of ROS & Gazebo
+
+If this is your first time using ROS and Gazebo, and you are not following
+specific instructions or a tutorials, we recommend using the latest long term
+support (LTS) versions of both ROS and Gazebo for the best user
+experience. Successive versions of ROS and Gazebo releases are named
+alphabetically, and the documentation for each version of ROS and Gazebo will
+indicate if it is an LTS version. It is worth noting that each version of ROS
+works best with a specific version of each Tier 1 platform. Tier one platforms
+are platforms / host operating systems that are used in the development of ROS
+and Gazebo.All of this information is outlined in
+[REP-2000](https://www.ros.org/reps/rep-2000.html). To summarize, the best user
+experience is to use the latest LTS version of ROS and the Tier 1 platform /
+operating system recommend for that version of ROS. If you host operating system
+does not match the Tier 1 operating system, consider using the Tier 1 platform
+in a virtual machine. This approach has the added benefit of not modifying your
+host OS and allowing you to roll back your mistakes.
+
+At the time of writing, our recommendation is that new users install:
+
+* [Ubuntu Jammy 22.04](https://www.releases.ubuntu.com/jammy/)
+* [ROS 2 Humble Hawksbill](https://www.ros.org/reps/rep-2000.html#humble-hawksbill-may-2022-may-2027)
+* [Gazebo Fortress](https://gazebosim.org/docs/fortress/install)
+
+# Summary of Compatible ROS and Gazebo Combinations
+
+This table includes all currently supported versions of ROS and Gazebo. All
+other ROS and Gazebo releases are end of life and we do not recommend their
+continued use.
+
+|                        	| **GZ Citadel (LTS)** 	| **GZ Fortress (LTS)** 	| **GZ Garden** 	|
+|------------------------	|----------------------	|-----------------------	|---------------	|
+| **ROS 2 Rolling**      	| X                    	| ✔️                     	| P             	|
+| **ROS 2 Humble (LTS)** 	| X                    	| ✔️                     	| P             	|
+| **ROS 2 Foxy (LTS)**   	| ✔️                    	| X                     	| X             	|
+| **ROS 1 Noetic (LTS)** 	| ✔️                    	| P                     	| X             	|
+
+
+* ✔️ - Recommended combination
+* X - Not recommended
```

Comment:
```suggestion
* X - Code incompatible
```
I think that we can go here with something stronger that "Not recommended" (which is fine just below in the P option). The X marks the options that we know they are not going to even compile since the code does not expect the given APIs.

Reply:
Maybe we say both. The goal is to make it clear that this is bad and dragons be here.

Reply:
fixed.